### PR TITLE
Feature/improve logs for grafana

### PR DESCRIPTION
**Pull request by julia-zack**
The pegin & pegout data processors were changed to add logs every time a peg status is updated.
It is important to can access them from grafana.

Some comments were also added with minor concerns.

### CODE REVIEW

File: src/services/pegin-data.processor.ts [hunk 1]
```diff
@@ -56,26 +56,51 @@ export class PeginDataProcessor implements FilteredBridgeTransactionProcessor {
         const rskReceiver = <string> lockBtcLog.arguments.receiver;
         status.rskRecipient = rskReceiver.toLowerCase();
       }
+      //
       status.status = RskPeginStatusEnum.LOCKED;
+
+      this.getLoggerContextInformation(status);
+      this.logger.trace(`[getPeginStatus] New PegIn locked with amount: ${lockBtcLog?.arguments.amount}.`);
+      // 60 & 63 lines shouldnt be inside the "if" condition?
```
Comment (lserra-iov) [inline src/services/pegin-data.processor.ts:64]: I think we could remove the if statement on L54 moving its content out.

File: src/services/pegout-data.processor.ts [hunk 2]
```diff
@@ -507,8 +539,13 @@ export class PegoutDataProcessor implements FilteredBridgeTransactionProcessor {
     return this.pegoutStatusDataService.set(pegout);
   }
```
Comment (lserra-iov) [inline src/services/pegout-data.processor.ts:540]: As the `getLoggerContextInformation` method is no longer returning a string this logs an undefined.

File: src/services/pegin-data.processor.ts [hunk 3]
```diff
@@ -56,26 +56,51 @@ export class PeginDataProcessor implements FilteredBridgeTransactionProcessor {
...
+      this.logPeginData(status);
+      this.logger.trace(`[getPeginStatus] New PegIn locked with amount: ${peginBtcLog.arguments.amount}.`);
+      
       return status;
     }
     if (this.hasThisLog(BRIDGE_EVENTS.REJECTED_PEGIN, extendedBridgeTx.events)) {
       const rejectedPeginLog: ExtendedBridgeEvent = extendedBridgeTx.events.find(event => event.name === BRIDGE_EVENTS.REJECTED_PEGIN) as ExtendedBridgeEvent;
       status.btcTxId = <string> rejectedPeginLog?.arguments.btcTxHash;
+
+      // TODO: this.logger.trace(`[getPeginStatus] New PegIn rejected with amount: ${amount}.`);
```
Comment (alexjavabraz) [inline src/services/pegin-data.processor.ts:88]: please remove this comments

File: src/services/pegin-data.processor.ts [hunk 4]
```diff
@@ -56,26 +56,51 @@ export class PeginDataProcessor implements FilteredBridgeTransactionProcessor {
...
+      this.logPeginData(status);
+      this.logger.trace(`[getPeginStatus] New PegIn rejected.`);
+      
       if (this.hasThisLog(BRIDGE_EVENTS.RELEASE_REQUESTED, extendedBridgeTx.events)) {
         status.status = RskPeginStatusEnum.REJECTED_REFUND;
+        // TODO: this.logger.trace(`[getPeginStatus] New PegIn rejected with amount: ${amount} will be refund.`);
```
Comment (alexjavabraz) [inline src/services/pegin-data.processor.ts:94]: please remove this comments

File: src/services/pegin-data.processor.ts [hunk 5]
```diff
@@ -56,26 +56,51 @@ export class PeginDataProcessor implements FilteredBridgeTransactionProcessor {
...
+        this.logPeginData(status);
+        this.logger.trace(`[getPeginStatus] New PegIn rejected will be refund.`);
         return status;
       }
       if (this.hasThisLog(BRIDGE_EVENTS.UNREFUNDABLE_PEGIN, extendedBridgeTx.events)) {
         status.status = RskPeginStatusEnum.REJECTED_NO_REFUND;
+        // TODO: this.logger.trace(`[getPeginStatus] New PegIn rejected with amount: ${amount} is unrefundable.`);
```
Comment (alexjavabraz) [inline src/services/pegin-data.processor.ts:101]: please remove this comments

File: src/services/pegout-data.processor.ts [hunk 6]
```diff
@@ -178,68 +187,71 @@ export class PegoutDataProcessor implements FilteredBridgeTransactionProcessor {
 
   }
 
-  private async processSignedStatus(extendedBridgeTx: ExtendedBridgeTx): Promise<void> {
-    const events: ExtendedBridgeEvent[] = extendedBridgeTx.events as ExtendedBridgeEvent[];
-    const releaseBTCEvent = events.find(event => event.name === BRIDGE_EVENTS.RELEASE_BTC);
+  // ----------------------------------------------------------pretty sure this function is not used anymore,
```
Comment (alexjavabraz) [inline src/services/pegout-data.processor.ts:190]: remove this comments please

File: src/services/pegout-data.processor.ts [hunk 7]
```diff
@@ -284,6 +296,10 @@ export class PegoutDataProcessor implements FilteredBridgeTransactionProcessor {
       newClonedPegoutStatus.batchPegoutIndex = index;
       newClonedPegoutStatus.batchPegoutRskTxHash = extendedBridgeTx.txHash;
 
+      this.logPegoutData(newClonedPegoutStatus);
+      this.logger.trace(`[processBatchPegouts] New PegOut waiting for confirmations
```
[the remark this answers is not in the document]
Comment (julia-zack) [reply src/services/pegout-data.processor.ts:300]: sorry, i dont understand.

**Review by lserra-iov — APPROVED** [review]

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**